BUFR_TableD_en_09.csv

```diff
@@ -152,6 +152,70 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 09,Vertical sounding sequences (conventional data),309023,(Single wavelength Elastic Backscatter Lidar sequence),Backscatter data section,033002,Quality information,,,Operational
 09,Vertical sounding sequences (conventional data),309023,(Single wavelength Elastic Backscatter Lidar sequence),Backscatter data section,010071,Vertical resolution,,,Operational
 09,Vertical sounding sequences (conventional data),309023,(Single wavelength Elastic Backscatter Lidar sequence),Backscatter data section,027079,Horizontal width of sampled volume,,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,301132,Common Header Sequence,,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,201151,Change data width (by YYY-128 bit places),,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,202130,Change scale (by YYY-128 places),,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,002121,Mean Frequency,,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,202000,Cancel scale change,,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,201000,Cancel data width change,,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,008021,Time significance,,,Operational
+09,Vertical sounding sequences (conventional data),309024,(Single wavelength wind profiler wind data sequence (product data)),,004025,Time period or displacement,,,Operational
```

Comment:
@amilan17 Note for " Time period or displacement" :  The time period indicates the duration in minutes over which the measurements have been averaged. The end of the time period is marked by the end of scan (see above).